pkg/apis/devops/v1alpha1/s2ibuildertemplate_types.go

```diff
@@ -57,7 +57,8 @@ type S2iBuilderTemplateStatus struct {
 // S2iBuilderTemplate is the Schema for the s2ibuildertemplates API
 // +k8s:openapi-gen=true
 // +kubebuilder:printcolumn:name="Framework",type="string",JSONPath=".spec.codeFramework"
-// +kubebuilder:printcolumn:name="BaseImage",type="string",JSONPath=".spec.baseImage"
+// +kubebuilder:printcolumn:name="BaseImages",type="string",JSONPath=".spec.baseImages"
```

Comment:
`baseImages` is an array, which is hard to print naturally. It is better to delete this line